packages/react-components/src/components/TypingIndicator.tsx

```diff
@@ -56,57 +147,79 @@ const getDefaultComponents = (
     const additionalCharCount =
       2 * (countOfUsersMentioned - 1) + (typingUser.displayName ? typingUser.displayName.length : 0);
     if (totalCharacterCount + additionalCharCount <= MAXIMUM_LENGTH_OF_TYPING_USERS || countOfUsersMentioned === 1) {
-      typingUsersMentioned.push(typingUser);
+      if (typingUser.displayName) {
+        typingUsersMentioned.push(typingUser);
+      }
       totalCharacterCount += additionalCharCount;
```

Comment:
Should these also be inside the `if (typingUser.displayName) {`
Not really sure if countOfUsersMentioned includes a user with no display name

Reply:
If we should not include users with no display name, might be simpler above this for loop to instead do:
```
const typingUsersWithNames = typingUsers.filter(user => !!user.displayName);
for (const typingUser of typingUsersWithNames) {
```